v1.1/recommended-production-settings.md

```diff
@@ -44,15 +44,15 @@ For details about controlling the number and location of replicas, see [Configur
 
 {% include faq/clock-synchronization.html %}
 
-## Cache Size
+## Cache and SQL Memory Size <span class="version-tag">Changed in v1.1</span>
 
-If you run multiple applications on the same machine as a CockroachDB node, you might consider manually setting the cache size instead of using the default 25% of available memory.
+By default, each node's cache and temporary SQL memory size is `128MiB` respectively. This default is optimal for running a local development cluster with multiple nodes on a single host. When running a production cluster with one node per host, however, these defaults may lead to out-of-memory errors. To prevent such errors and improve read performance, it's recommended to manually increase the cache and SQL memory size to 25% or more of available system memory.
```

Comment:
I shy away from using the term `optimal`. Perhaps `This default is reasonable for running...`.

I think it is worthwhile to distinguish the effects of the flags. `--cache` will improve read performance. `--max-sql-memory` will increase the number of concurrent SQL queries that can be processed and the complexity of the queries that can be processed.

Cc @knz